src/createAction.js

```diff
@@ -13,7 +13,7 @@ export default function createAction(type, payloadCreator, metaCreator) {
     };
 
     const payload = hasError ? args[0] : finalPayloadCreator(...args);
-    if (!(payload === null || payload === undefined)) {
+    if (payload !== undefined) {
```

Comment:
Sure, but just consider that it does exactly the same thing: https://github.com/lodash/lodash/blob/master/lodash.js#L11878